CONTRIBUTING.rst

```diff
@@ -38,9 +38,18 @@ If you are proposing a feature:
 Chat with other developers
 ==========================
 
-You can join the official #aboutcode IRC channel on freenode
-(server chat.freenode.net) to chat about scancode and other related tools.
-You can use your favorite IRC client or use the web chat at https://webchat.freenode.net/
+For other questions, discussions, and chats, we have:
+
+- a mailing list at https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss
+
+- an official Gitter channel at https://gitter.im/aboutcode-org/discuss
```

Comment:
I would put Gitter first, and de-emphasize IRC and the mailing list IMHO.

Reply:
I picked it up from https://github.com/nexB/scancode-toolkit#support to maintain consistency 😛 . No problem, I'll change it in a minute 😉